static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -8,6 +8,9 @@ import mediator from 'lib/mediator';
 import { local } from 'lib/storage';
 import { mergeCalls } from 'common/modules/async-call-merger';
 import { getUrlVars } from 'lib/url';
+import fetch from 'lib/fetch-json';
+
+const qs = require('qs');
```

Comment:
could this be `import qs from 'qs';` instead?

Reply:
done